Schedules.API/Tasks/Db.cs

```diff
@@ -15,6 +15,8 @@ public static class Db
       var providerName = EnvironmentVariableHelper.GetEnvironmentVariable(providerKey);
       var connectionString = EnvironmentVariableHelper.GetEnvironmentVariable(connectionKey);
 
+     
```

Comment:
Please remove the changes from this file. White space changes aren't important to this update.